examples/CopyWorkspaceSampleScript.ps1

```diff
@@ -0,0 +1,190 @@
+﻿# PART 1: Authentication
+# ==================================================================
+ Login-PowerBI
+
+# PART 2: Getting source and target workspace
+# ==================================================================
+# STEP 2.1: Get the source workspace
+$source_group_ID = ""
+while (!$source_group_ID) {
+    $source_group_name = Read-Host -Prompt "Enter the name of the workspace you'd like to copy from"
+
+    if($source_group_name -eq "My Workspace") {
+        $source_group_ID = "me"
+        break
+    }
+
+    $group = Get-PowerBIWorkspace -Name $source_group_name -ErrorAction SilentlyContinue
+    if (!$group -or $group.isReadOnly -eq "True") {
+        "Invalid choice: you must have edit access to the workspace"
+        break
+    } else {
+        $source_group_ID = $group.id
+        break
+    }
+
+    if(!$source_group_ID) {
+        "Please try again, making sure to type the exact name of the workspace"  
+    } 
+}
+
+# STEP 2.2: Get the target workspace
+$target_group_ID = "" 
+while (!$target_group_ID) {
+    $target_group_name = Read-Host -Prompt "Enter the name of the workspace you'd like to copy to"
+
+    $target_group = Get-PowerBIWorkspace -Name $target_group_name -ErrorAction SilentlyContinue
+    if(!$target_group) {
+        $target_group = New-PowerBIWorkspace -Name $target_group_name -ErrorAction SilentlyContinue
```

Comment:
I think we should take it as a parameter, whether user want to create a new workspace.